edweiss-app/app/(app)/lectures/remotecontrol/index.tsx

```diff
@@ -53,12 +53,14 @@ const RemoteControlScreen: ApplicationRoute = () => {
     Voice.onSpeechResults = (res) => { setTalked(res.value ? res.value[0] : ""); };
 
     return (
-        <AbstractRmtCrl
-            handleRight={() => handleRight(isRecording, currentPage, totalPages, setIsRecording, setCurrentPage, stopRecording)}
-            handleLeft={() => handleLeft(isRecording, currentPage, setIsRecording, setCurrentPage, stopRecording)}
-            handleMic={() => handleMic(isRecording, setIsRecording, startRecording, stopRecording)}
-            isRecording={isRecording}
-        />
+        <>
+            {<AbstractRmtCrl
```

Comment:
A small reformat might be needed, but yeah surrounding the component within `<></>` may have done the trick. When I look back to it I realize that I may have to put them in the `todo/create` and `todo/edit` screens maybe 😁.